policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -435,71 +440,65 @@ impl ServiceRoutes {
                 policy.opaque = self.opaque;
                 modified = true;
             }
+            if self.accrual != policy.accrual {
+                policy.accrual = self.accrual.clone();
+                modified = true;
+            }
             modified
         });
     }
 }
 
-// TODO: could bubble up errors through `Result`
 fn parse_accrual_config(
     annotations: &std::collections::BTreeMap<String, String>,
 ) -> Result<Option<FailureAccrual>> {
-    match annotations.get("balancer.linkerd.io/mode") {
-        Some(v) if v == "consecutive" => {
-            // TODO (matei): value is kind of out of pocket, how do we establish what's good
-            // here.
-            let max_failures = annotations
-                .get("balancer.linkerd.io/failure-accrual-consecutive-failures")
-                .map(|s| {
-                    s.parse::<u32>()
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                }) // Option<Result<>
-                .transpose()?
-                .unwrap_or(7);
-
-            let max_penalty = annotations
-                .get("balancer.linkerd.io/max-penalty")
-                .and_then(|s| {
-                    parse_duration(s)
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                })
-                .unwrap_or_else(|| time::Duration::from_secs(60));
-
-            let min_penalty = annotations
-                .get("balancer.linkerd.io/min-penalty")
-                .and_then(|s| {
-                    parse_duration(s)
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                        .ok()
-                })?
-                .unwrap_or_else(|| time::Duration::from_secs(1));
-            let jitter = annotations
-                .get("balancer.linkerd.io/jitter")
-                .and_then(|s| {
-                    s.parse::<f64>()
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                        .ok()
-                })
-                .unwrap_or_default();
-            if min_penalty > max_penalty {
-                tracing::debug!(?min_penalty, ?max_penalty, "Invalid backoff configuration");
-                return Some(FailureAccrual::Consecutive {
+    annotations
+        .get("balancer.linkerd.io/mode")
+        .map(|mode| {
+            if mode == "consecutive" {
+                let max_failures = annotations
+                    .get("balancer.linkerd.io/failure-accrual-consecutive-failures")
+                    .map(|s| s.parse::<u32>())
+                    .transpose()?
+                    .unwrap_or(7);
+
+                let max_penalty = annotations
+                    .get("balancer.linkerd.io/max-penalty")
+                    .map(|s| parse_duration(s))
```

Comment:
style nit, TIOLI: could just be
```suggestion
                    .map(parse_duration)
```